src/containers/HelpMessage.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { getUserLocation } from '../actions/index';
+import HelpMessage from '../components/HelpMessage';
+
+const mapStateToProps = state => ({
+  userLocation: state.userLocation,
+});
+
+const mapDispatchToProps = dispatch => ({
+  // registerUserLocation: () => dispatch(registerUserLocation()),
```

Comment:
mapDispatchToProps is empty. You can remove mapDispatchToProps itself.

Reply:
i will need them, once i can connect to the backend.

Reply:
Got it.